drillsrs/cmd/review.py

```diff
@@ -123,10 +129,13 @@ def decorate_arg_parser(self, parser: argparse.ArgumentParser) -> None:
         parser.add_argument(
             '-n', type=int, default=None,
             help='set max number how many flashcards to review')
+        parser.add_argument(
+            '-m', '--mode', type=str, default="direct",
+            help='mode: direct, reversed, mixed')
```

Comment:
Please use `enum` to store possible choices:

```python3
class Mode(enum.Enum):
    direct = 'direct'
    reversed = 'reversed'
    mixed = 'mixed'
```

then add `choices` kwarg here to prevent the user from passing nonsense option.
Naturally replace all occurrences of magic strings to use the new enum.